src/WebJobs.Script/Rpc/ILanguageWorkerChannel.cs

```diff
@@ -19,9 +19,9 @@ public interface ILanguageWorkerChannel
 
         void SetupFunctionInvocationBuffers(IEnumerable<FunctionMetadata> functions);
 
-        void SendFunctionLoadRequests();
+        Task LoadFunctionsAsync(IEnumerable<FunctionMetadata> functions = null);
```

Comment:
Rename SendFunctionLoadRequestsAsync as this method is actually sending load requests